code/modules/vehicles/vehicle_actions.dm

```diff
@@ -191,3 +191,38 @@
 			owner.say("Thank you for the fun ride, [clown.name]!")
 			last_thank_time = world.time
 			C.ThanksCounter()
+
+/datum/action/vehicle/ridden/scooter/skateboard/ollie
+	name = "Ollie"
+	desc = "Get some air! Land on a table to do a gnarly grind."
+	button_icon_state = "skateboard_ollie"
+	var/last_ollie
+
+/datum/action/vehicle/ridden/scooter/skateboard/ollie/Trigger()
+	if(world.time - last_ollie > 5)
+		var/obj/vehicle/ridden/scooter/skateboard/V = vehicle_target
+		if (V.grinding)
+			return
+		var/mob/living/L = owner
+		var/turf/landing_turf = get_step(V.loc, V.dir)
+		L.adjustStaminaLoss(V.instability*2)
+		if (L.getStaminaLoss() >= 100)
+			playsound(src, 'sound/effects/bang.ogg', 20, 1)
+			V.unbuckle_mob(L)
+			L.throw_at(landing_turf, 2, 2)
+			V.visible_message("<span class='danger'>[L] misses the landing and falls on [L.p_their()] face!</span>")
+		else
+			L.spin(4, 1)
+			animate(L, pixel_y = -6, time = 4)
+			animate(V, pixel_y = -6, time = 3)
+			playsound(V, 'sound/vehicles/skateboard_ollie.ogg', 50, 1)
+			L.pass_flags |= PASSTABLE
+			V.pass_flags |= PASSTABLE
+			L.Move(landing_turf, vehicle_target.dir)
+			L.pass_flags &= ~PASSTABLE
+			V.pass_flags &= ~PASSTABLE

```

Comment:
You should check if either had the flag before already. Otherwise stuff that should have the flag suddenly loses it.

Reply:
I added this for the mob, for the vehicle should be unnecessary for the foreseeable future

Reply:
if there are multiple sources you're better off converting to a trait that adds or removes the flag

Reply:
you mean turning PASSTABLE into a trait? I wouldn't even know where to start, to be quite honest

Reply:
This would indeed break the passtable gained from Dwarfism. An easy but hacky fix would be to check if they already have the flag further up and skip assigning/unassigning the flags if they do. Also I didn't look very hard but I don't see this flag ever get unassigned

Reply:
that's exactly what I did, this is outdated

This flag also gets assigned/unassigned in wings code, so I'm guessing dwarves with wings have this exact issue

Part of the issue is that mobs buckled to vehicles do not inherit their flags, which I think would make sense and wouldn't require workarounds like these

Reply:
stop it, that's a hack, make a goddamn trait that applies the flag whenever there's at least once source and removes it again when there are none left

REEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEeee

Reply:
done did it (I think)

thanks for the help orange man